dynamiqs/mesolve/mesolve.py

```diff
@@ -92,6 +93,7 @@ def mesolve(
 
     # convert t_save to a tensor
     t_save = torch.as_tensor(t_save, dtype=options.rdtype, device=options.device)
+    check_time_array(t_save, 't_save')
```

Comment:
```suggestion
    check_time_array(t_save, error_str='t_save')
```
So it reads without having to check the function. Can be `error_str`, `arg_name`, `raise_str`, `str` or whatever you prefer.
Also applies to other calls.